homework/hw01_sachin_tolstoy.ipynb

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
-2 -- instructions were explicit about no ipynb

Reply:
Hi Professor, 

I would like to clarify that I had mailed you during the submission of this assignment that I was unwell with Covid during the first and second week of class and apart from that I was finding it hard to jostle with Github since it was my first time with this application. Can you please kindly reconsider my assignment and I will be sure to submit all the assignments on time. Thank you very much.

Reply:
That's right-- I see now that you did email me your assignment on time.  I will still take off two points for not submitting via Github on time.
6/10